apps/docs/README.md

```diff
@@ -1,36 +1,11 @@
-# Website
+# StyleX website
```

Comment:
The deployment instructions are also probably wrong, but there are no other instructions on how deploys work.